pkg/build/controller/strategy/custom.go

```diff
@@ -131,8 +131,8 @@ func (bs *CustomBuildStrategy) CreateBuildPod(build *buildv1.Build) (*corev1.Pod
 
 	if strategy.ExposeDockerSocket {
 		setupDockerSocket(pod)
```

Comment:
We're continuing to support docker-driven builds for the Custom strategy?

Reply:
for now i'm intending to leave the mount in place.  If your host node has a /var/lib/docker.sock, then great, your custom builds keep working the way they didn't.  If it doesn't, they don't (and there's nothing we can do about it)